source/coffee/image-picker.coffee

```diff
@@ -64,6 +64,16 @@ class ImagePicker
         option.mark_as_selected()
       else
         option.unmark_as_selected()
+    jQuery("li", @picker).each () ->
+      if jQuery(this).hasClass("selected") && jQuery("li", @picker).find(":focus").length == 0
+        jQuery(this).prop("tabindex", "0");
+      else
+        jQuery(this).prop("tabindex", "-1");
+      return
+    if jQuery(this).find(":selected").length == 0 && jQuery("li", @picker).find(":focus").length == 0
+      $("li:first-child", @picker).find(".selectable").prop("tabindex", "0");
+    return
```

Comment:
Should these go 0, 1, 2, 3, etc? Or is it ok that they are all 0?

Reply:
This is okay that they are all 0, means they are all 'tabbable' at the same priority, which then uses the order in the DOM to iterate through

Reply:
Awesome, this is looking great!